src/main/webapp/app/exam/participate/exercises/modeling/modeling-exam-submission.component.html

```diff
@@ -1,7 +1,11 @@
 <jhi-resizeable-container class="col-12" *ngIf="exercise">
     <!--region Left Panel-->
     <span left-header *ngIf="exercise.title">{{ exercise.title }}</span>
-    <span left-header>&nbsp;[{{ exercise.maxScore }} {{ 'artemisApp.examParticipation.points' | translate }}]</span>
+    <span left-header
+        >&nbsp;[{{ exercise.maxScore }} {{ 'artemisApp.examParticipation.points' | translate }}
```

Comment:
Can we move the closing tags onto the same line? (Here and in the files below.) Imho
```
<span
    >
```
looks very weird compared to just having `<span>` when there are no attributes.